.gitignore

```diff
@@ -12,6 +12,8 @@ doc/neddoc
 
 /out
 
+/.settings
```

Comment:
These changes might be incidental - at least it is not mentioned in the commit message. I guess it should be safe to remove, but wanted to confirm with you first

Reply:
Yes, you're right. Safe to remove. _Built with the omnet++ IDE during development, then forgot to delete it._